HTML pages/home.html

```diff
@@ -0,0 +1,218 @@
+    <link rel="stylesheet" href="/Stylesheets pages/featured.css" />
+    <link rel="stylesheet" href="/Stylesheets pages/menu.css" />
+  </head>
+  <body id="homepage">
+    <header>
+      <div class="topbar">
+        <div class="topbar-content">
+        <img src="/assets/img/facebook.png" alt="fb-Icon" class="fb">
+        <img src="/assets/img/twitter.png" alt="tweet icon" class="twt">
+      </div>
+        <div class="links-topbar">
+        <a href="/">My Page</a>
+        <a href="/">Logout</a>
+      </div>
+      </div>
+
+      <div class="desktop-header-nav">
+        <nav class="header-nav-d">
+          <a href="home.html" class="logo-desktop">
+            <h3>GYMSHRK</h3>
+          </a>
+          <div class="header-nav-desktop">
+            <ul class="menu-list">
+              <li class="menu-item">
+                <a href="home.html" class="link-portfolio"> Home </a>
+              </li>
+              <li class="menu-item">
+                <a href="about.html" class="link-about"> About </a>
+              </li>
+              <li class="menu-item">
+                <a href="/" class="link-contact"> Contact </a>
+              </li>
+            </ul>
+          </div>
+          <div class="reg-btn">
+            <button type="button">RG Gymshrk</button>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <!-- First Section -->
+
+    <section class="first-section">
+       <div class="header-nav-mobile">
+        <button type="button" class="menu-bar-mobile" class="">
+          <img src="/assets/Icon - Menu.svg" alt="mobile menu bar icon" />
+        </button>
+
+        <!-- close button of mobile menu -->
+
+        <button type="button" class="menu-bar-close">
+          <img src="/assets/Icon-Cancel.svg" alt="Close button" />
+        </button>
+
+        <!--overlay background mobile menu function -->
+
+        <div class="overlay">
+          <nav class="navigation-mobile">
+            <div class="navigation">
+              <ul class="menu-list-mobile">
+                <li class="menu-item-mobile">
+                  <a href="home.html" class="link-home"> Home </a>
+                </li>
+                <li class="menu-item-mobile">
+                  <a href="about.html" class="link-about"> About </a>
+                </li>
+                <li class="menu-item-mobile">
+                  <a href="/" class="link-contact"> Contact </a>
+                </li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+      </div>
+
+      
+      <div class="flex-container">
+        <div class="home-heading">
+        <h2 class="small-title">Hello Sharing World!</h2>
+        <h1 class="big-title">GYMSHRK Global Fitness Competition<br>2023</h1>
+        </div>
+       <div class="heading-box">
+        <p class="head-box-txt">Lorem Ipsum is simply dummy text of the printing and typesetting.
+          Lorem Ipsum is simply dummy text of the printing and typesetting.
```

Comment:
Weldone in making the required changes as suggested by the previous reviewer, please try to make your design to look as close as possible to the design given, by making the background color of this highlighted section to be lighter and closely similar to the main background color as it is in the design given to help us a better UI that is more presentable 😄 

Design given :
![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/20537712/7740c843-6ae7-4542-b939-d0b2f8bfac33)

Your design :
![image](https://github.com/Rifat8080/mahadi_capstone_project_1/assets/20537712/5e94d75c-4929-4825-ac42-6c532f17ba20)